src/AboutPage/About.tsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function About() {
   return (
-    <div>About</div>
+    <body>
+      <section className='flex justify-center items-center h-screen w-screen flex-col'>
+        <h1 className=' text-5xl m-10'>About</h1>
+        <img src="/src/images/hero-img.png" className='w-5/6 h 3/6'></img>
+      </section>
+
+      <section className='flex flex-col justify-center items-center'>
+        <article className='flex flex-row w-5/6 m-20'>
+          <div className='flex flex-col align-left '>
+            <h2 className='text-sm'>About us</h2>
+            <h3 className='text-3xl'>Our Restaurant</h3>
+            <p className=''>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Officiis fuga saepe eligendi nobis? Odio necessitatibus id deleniti exercitationem qui quaerat sapiente incidunt neque vitae vero, quia, fugit, quo optio obcaecati!</p>
+          </div>
+          <img src="/src/images/IMG_1053.png" className='w-3/6'></img>
```

Comment:
These images should have better names so that if we replace the image, you don't have to change it in this file. For example, something like `about-img-1.png` and `about-img-2.png`.